waspc/packages/wasp-config/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "target": "es2022",
+    "allowJs": true,
+    "resolveJsonModule": true,
+    "moduleDetection": "force",
+    "isolatedModules": true,
+
+    // linting
+    "strict": true,
+    "noUncheckedIndexedAccess": true,
+    "noImplicitOverride": true,
+
+    "module": "NodeNext",
+    "outDir": "dist",
+    "declarationMap": true,
+
+    "declaration": true,
+    "lib": ["es2022"]
+  },
+   // better structure in output
```

Comment:
Maybe you could elaborate this one a bit more.

Reply:
I shouldn't have actually even added this comment. This option just points to "where the source is", so this is a completely normal use case. 

I've now removed it.